tests/ovs_test.go

```diff
@@ -35,4 +38,52 @@ var _ = Describe("ovs-cni", func() {
 			})
 		})
 	})
+
+	Describe("pod ovs-bridge connectivity tests", func() {
+		const bridgeName = "br-test"
+		Context("When creating ovs-bridge on the host node", func() {
+			BeforeEach( func() {
+				cluster.AddOvsBridgeOnNode(bridgeName)
+			})
+			AfterEach( func() {
+				cluster.RemoveOvsBridgeOnNode(bridgeName)
+			})
+
+			Context("and adding ovs network with vlan", func() {
```

Comment:
"and a network attachment definition is defined"

Reply:
Done